lib/Screens/Tasks/completeHealthCheck.dart

```diff
@@ -683,7 +707,15 @@ class _TaskHealthCheck extends State<TaskHealthCheck> {
                                     style: TextStyle(fontSize: 20),
                                   ),
                                   onPressed: () {
-                                    _onSave(taskObserver);
```

Comment:
Passing in the task observer was giving weird results, so I just did those operations right in the on pressed method instead